examples/template.html

```diff
@@ -124,6 +124,10 @@
             animate();
           });
 
+          $(document).ready(function(){
```

Comment:
This is something that I've been experimenting with.

`$(document).ready(function()){}` seems to be called when all of the visible elements are loaded.
This actually loads after `$(function()){}` is called.

I am not sure why this is the case - it seems counterintuitive, especially considering that they are supposed to be equivalent (at least according to SO).

I encourage you to test this out.